src/playbacks/html5_video/html5_video.js

```diff
@@ -215,7 +240,10 @@ export default class HTML5Video extends Playback {
   // On mobile device, HTML5 video element "retains" user action consent if
   // load() method is called. See Player.consent().
   consent() {
-    !this.isPlaying() && this.el.load()
+    if (!this.isPlaying()) {
+      this.el.load()
+      super.consent()
```

Comment:
I'd just move the `super.consent()` call to before the load call to prevent the `consented` state from being inconsistent during event handling.